src/Symfony/Component/Yaml/Parser.php

```diff
@@ -160,6 +160,12 @@ private function doParse(string $value, int $flags)
 
             Inline::initialize($flags, $this->getRealCurrentLineNb(), $this->filename);
 
+            if (preg_match('/([\x00-\x08\x0B-\x0C\x0E-\x1F\x7F])/', $this->currentLine, $matches)) {
+                @trigger_error(sprintf('Parsing YAML strings that contain the "%s" unicode sequence is deprecated since Symfony 4.3 and will throw a ParseException in 5.0.', strlen($matches[1])), E_USER_DEPRECATED);
+                // to be thrown in 5.0
+                //throw new ParseException(sprintf('Unexpected character 0x%X', $matches[1]));
```

Comment:
we should pass `int $parsedLine = -1, string $snippet` right

Reply:
oh ha! these are the sprintf argument :D never mind then.

Unless we should still provide the current line of course :)